fns-hl7-pipeline/fn-receiver-debatcher/function/extensions.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
is this file needed? we're not using dotnet or c#

Reply:
deleted - added to gitignore